manual-tests.md

```diff
@@ -49,3 +49,108 @@
 2. Select a project to build.
 3. Run `snapcraft cleanbuild --remote <remote>` where `<remote>` is
    the name you gave the remote on step 1.
+
+# Test the PC kernel.
+
+1. Get the PC kernel source:
+
+    $ git clone -b pc https://git.launchpad.net/~ubuntu-kernel/ubuntu/+source/linux-snap/+git/xenial
```

Comment:
hmm, this kernel snap does not use the kernel plugin, we can still use it but meh
https://git.launchpad.net/~ubuntu-kernel/ubuntu/+source/linux-snap/+git/xenial/tree/snapcraft.yaml?h=pc